docs/computing/quantum-computing/helmi/running-on-helmi.md

```diff
@@ -1,46 +1,46 @@
 # Running on Helmi
 
 !!! info "Give feedback!"
-	**All feedback is highly appreciated**, please comment on your
-	experience to [fiqci-feedback@postit.csc.fi](mailto:fiqci-feedback@postit.csc.fi).
+    **All feedback is highly appreciated**, please comment on your
+    experience to [fiqci-feedback@postit.csc.fi](mailto:fiqci-feedback@postit.csc.fi).
 
 ## Running Jobs
 
-Jobs can be submitted to the `q_fiqci` queue by specifying `--partition=q_fiqci` in batch scripts. 
+Jobs can be submitted to the `q_fiqci` queue by specifying `--partition=q_fiqci` in batch scripts.
 
-Helmi currently supports submitting jobs using Qiskit or Cirq. Qiskit and Cirq scripts can only be submitted as ordinary python files. 
+Helmi currently supports submitting jobs using Qiskit or Cirq. Qiskit and Cirq scripts can only be submitted as ordinary python files.
 To submit and run jobs on Helmi you need to use the correct environment on LUMI.
 
-* First, run `module use /appl/local/quantum/modulefiles`. The available modules will now show up on `module avail`. 
+* First, run `module use /appl/local/quantum/modulefiles`. The available modules will now show up on `module avail`.
 * Second, depending on if you want to use the Qiskit or Cirq environment, run:
-	* `module load helmi_qiskit` or 
-	* `module load helmi_cirq`
+    * `module load helmi_qiskit` or
+    * `module load helmi_cirq`
 
 
-`helmi_qiskit` and `helmi_cirq` provide pre-made python environments to directly run on Helmi. 
-If you wish to add your own python packages to the pre-made python environment you can do so with `python -m pip install --user package`. 
+`helmi_qiskit` and `helmi_cirq` provide pre-made python environments to directly run on Helmi.
+If you wish to add your own python packages to the pre-made python environment you can do so with `python -m pip install --user package`.
 
 
 !!! info "Creating your own python environment"
-	Users can create their own python environment if they wish. The only prerequisite is to load the `helmi_standard` module. 
-	To create your own environment the [container wrapper tool](https://docs.lumi-supercomputer.eu/software/installing/container-wrapper/) is recommended.
+    Users can create their own python environment if they wish. The only prerequisite is to load the `helmi_standard` module.
+    To create your own environment the [container wrapper tool](https://docs.lumi-supercomputer.eu/software/installing/container-wrapper/) is recommended.
 
 The current supported software versions on helmi are:
 
-```bash
-Cirq on IQM 	cirq_iqm 	>= 13.0, < 14.0
-Qiskit on IQM 	qiskit_iqm 	>= 12.0, < 13.0
-IQM client 	iqm_client 	>= 16.0, < 17.0
-Cortex CLI 	iqm_cortex_cli 	>= 5.8, < 6.0
+```text
+Cirq on IQM    cirq_iqm        >= 14.0, < 15.0
+Qiskit on IQM  qiskit_iqm      >= 13.0, < 14.0
+IQM client     iqm_client      >= 17.1, < 18.0
+Cortex CLI     iqm_cortex_cli  >= 5.8, < 6.0
 ```
```

Comment:
```suggestion
| Software | Module | Versions |
|----------|--------|----------|
| Cirq on IQM | cirq_iqm | >= 14.0, < 15.0 |
| Qiskit on IQM | qiskit_iqm | >= 13.0, < 14.0 |
| IQM client | iqm_client | >= 17.1, < 18.0 |
| Cortex CLI | iqm_cortex_cli | >= 5.8, < 6.0 |
```

I'd suggest, if I may, using a table for the supported versions. I just don't know what to call the middle column. I titled it "Module" here as a placeholder, but I don't know what the entries are.